### PR TITLE
ISPN-10309 Convert usages of size to use sizeAsync

### PR DESCRIPTION
**Pull request by wburns**
https://issues.jboss.org/browse/ISPN-10896

### CODE REVIEW

**Comment by gustavonalle** [comment]
Since you are at this, there is this one to convert too:  https://github.com/infinispan/infinispan/blob/48cef5884866ea2c50f6c5b6ae43101ff10cd698/server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java#L230

**Comment by wburns** [comment]
> Since you are at this, there is this one to convert too:
> 
> https://github.com/infinispan/infinispan/blob/48cef5884866ea2c50f6c5b6ae43101ff10cd698/server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java#L230

Yes, for sure. IntelliJ didn't find that occurrence :)

**Comment by wburns** [comment]
Oh actually I did convert it. I just forgot to stage the commit, oops.

File: server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java [hunk 1]
```diff
@@ -191,36 +189,40 @@ public Invocations getInvocations() {
...
+                        cacheInfo.transactional = cacheConfiguration.transaction().transactionMode().isTransactional();
+                        cacheInfo.persistent = cacheConfiguration.persistence().usingStores();
+                        cacheInfo.bounded = cacheConfiguration.expiration().maxIdle() != -1 ||
+                              cacheConfiguration.expiration().lifespan() != -1;
+                        cacheInfo.secured = cacheConfiguration.security().authorization().enabled();
+                        cacheInfo.indexed = cacheConfiguration.indexing().index().isEnabled();
+                        cacheInfo.hasRemoteBackup = cacheConfiguration.sites().hasEnabledBackups();
+                        return cacheInfo;
+                        // Only request 10 caches sizes in parallel
+                     }), 10)
```
Comment (ryanemerson) [inline server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java]: Any rationale behind 10?

**Comment by gustavonalle** [comment]
merged, thanks @wburns !
(I assumed @ryanemerson question was not a show stopper to merge)

**Comment by ryanemerson** [comment]
Nope, was curious as much as anything.

File: server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java (hunk 1 above)
Comment (wburns) [reply server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java]: Just wanted to limit it to something lower. RxJava2 would use 128 by default and that seems like way too many. To be honest I am debating if we want to lower it to 1 even.

Comment (danberindei) [reply server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java]: +1 to use limit it to 1

Comment (wburns) [reply server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java]: I have created https://github.com/infinispan/infinispan/pull/7568

**Pull request merged**